src/client/containers/404Page/404Page.css

```diff
@@ -0,0 +1,83 @@
+.body-background-image {
+  background-image: url(../../assets/images/Vector.svg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+.center-container {
+  width: 45%;
+  height: 65%;
+  margin: 9% auto;
+  background: rgba(179, 64, 74, 0.73);
+  position: relative;
+  font-family: 'Roboto Mono', monospace;
+}
+
+.center-container img {
+  margin: 1rem 1rem;
+  width: 7rem;
+  height: 7rem;
+}
+
+.main-text {
+  position: absolute;
+  top: 25%;
+  left: 33%;
```

Comment:
Be careful with absolute position that should be your last way out to style it because it will in a lot of case not be very responsive for different screen sizes